cfgov/alerts/post_sqs_messages.py

```diff
@@ -69,9 +89,35 @@
         MaxNumberOfMessages=10,
     )
 
+    # Initialize github API & repo to post to
+    gh = github3.login(
+        token=args.github_token,
+        url=args.github_url,
+    )
+    repo = gh.repository(
+        args.github_user,
+        args.github_repo,
+    )
+
     if 'Messages' in response:
-        # Post these messages to specified Github repo
         for message in response['Messages']:
             body = message['Body']
+            title = body.split(" - ")[0]
             logger.info('Retrieved message {} from SQS'.format(body))
-            repo.create_issue(title=body[:20], body=body)
+
+            issue = matching_issue(title=title, issues=repo.iter_issues())
+            if issue:
+                # Issue already exists,
+                # add comment to it to document it happened again
+                issue.create_comment(body=body)
+            else:
+                # New issue, post to github
+                issue = repo.create_issue(title=title, body=body)
+                # AND post to chat, if credentials provided
+                if args.mattermost_webhook_url and args.mattermost_username:
+                    post_to_chat(
+                        endpoint=args.mattermost_webhook_url,
+                        username=args.mattermost_username,
+                        message=body,
+                        issue_url=issue.html_url,
+                    )
```

Comment:
Will this post to Mattermost with every SQS alert, or just with ones that create new issues? 

I think we should post to Mattermost for every alert... our when-to-send-alerts policy should ensure that we're not spamming the channel.